hasher-matcher-actioner/terraform/api-surface/main.tf

```diff
@@ -0,0 +1,205 @@
+# Copyright (c) Facebook, Inc. and its affiliates. All Rights Reserved
```

Comment:
Yea... another naming comment! Why api-surface vs just api? For webapp I used webapp not webapp-surface or ui-surface. Seems like we need to pick one.